listenbrainz_spark/path.py

```diff
@@ -1,12 +1,34 @@
 import os
+import config
+
+# Parent directory containing listens and data driven from ListenBrainz.
+LISTENBRAINZ_DATA_DIRECTORY = os.path.join('/', 'data', 'listenbrainz')
+# Directory containing similar artist relation.
+# (This is a temporary path till incremental dumps for similar artists are prepared)
+SIMILAR_ARTIST_DIR = os.path.join('/', 'data', 'listenbrainz', 'similar_artists')
```

Comment:
As param pointed out, we should use the LISTENBRAINZ_DATA_DIRECTORY for anything that goes under /data/listenbrainz so that if we need to move it, we can change one value only.